aria/modeling/types.py

```diff
@@ -177,6 +177,7 @@ def coerce(cls, key, value):
         except ValueError as e:
             raise exceptions.StorageError('SQL Storage error: {0}'.format(str(e)))
```

Comment:
i wonder if StorageErrors should be named something different now that the package is named "modeling"